scripts/asset-import/src/defaultConfig.js

```diff
@@ -1,4 +1,5 @@
 module.exports = {
+    assetDirectory: '_Deprecated',
     branch: 'master',
     workingDir: '/tmp',
     destinationDir: 'src',
```

Comment:
nit: the other directory names are shortened to `Dir`

```suggestion
    assetDir: '_Deprecated',
    branch: 'master',
    workingDir: '/tmp',
    destinationDir: 'src',
```